src/components/dataDisplay/dataResults.js

```diff
@@ -1,5 +1,5 @@
 import { Box, Typography, useTheme, useMediaQuery } from "@mui/material";
-import MotifDisplay from "components/custom/MotifDisplay";
```

Comment:
If it's no longer required, can you remove the MotifDisplay from the project?

Reply:
Resolved in latest commit. 
I also removed the MotifItem component, since that's no longer used either